docs/data/tree-view/tree-item-customization/useTreeItemHookPublicAPI.tsx

```diff
@@ -34,9 +41,21 @@ const CustomTreeItem = React.forwardRef(function CustomTreeItem(
   props: TreeItemProps,
   ref: React.Ref<HTMLLIElement>,
 ) {
-  const { publicAPI } = useTreeItem(props);
+  const { publicAPI, status } = useTreeItem(props);
```

Comment:
unrelated to this PR, but I just realized we can do the same thing with `useTreeItemUtils` too 🤔 Could become confusing for user

Reply:
True, I think it will be solved when we migrate the Tree Item to the Base UI DX since `useTreeItem` will go away.